internal/common/tsconfig.bzl

```diff
@@ -54,6 +54,9 @@ def create_tsconfig(
     Returns:
       A nested dict that corresponds to a tsconfig.json structure
     """
+    if (type(files) != type(depset())):
+      fail("expected files argument to be a depset, got", type(files))
```

Comment:
nit: the second parameter of fail() is supposed to be the attribute that caused the error:
https://docs.bazel.build/versions/master/skylark/lib/globals.html#fail